govcr.go

```diff
@@ -28,12 +30,21 @@ const defaultCassettePath = "./govcr-fixtures/"
 type VCRConfig struct {
 	Client            *http.Client
 	ExcludeHeaderFunc ExcludeHeaderFunc
+
+	// Deprecated, use RequestFilter
```

Comment:
If we're going to go down this road, let's get rid off `ResponseFilterFunc` altogether.
This would form v3 of govcr.